# Publication Revision Update — August 23, 2026

This notebook consolidates the figure and table updates produced during the current revision round. The six earlier publication notebooks are retained as a historical reproducibility record; this notebook identifies the revised artifacts that supersede their corresponding presentation or diagnostic definitions.

Key terminology is standardized to **constant $u_0$** and **constant $U_{\max}=30$ m s$^{-1}$**. Cyclone and anticyclone feature diagnostics use the eddy surface-pressure field where specified. No manuscript or Word document is generated or edited by this notebook.

In [1]:
from pathlib import Path
import pandas as pd
from IPython.display import Image, display

ROOT = Path.cwd()
if not (ROOT / 'publication_notebooks').exists():
    ROOT = ROOT.parent
ASSET_ROOT = ROOT / 'publication_notebooks' / 'revision_20260823'
manifest = pd.read_csv(ASSET_ROOT / 'revision_manifest.csv')
manifest[['label', 'title', 'scientific_or_presentation_change']]

,label,title,scientific_or_presentation_change
0,Current Figure 6,RWB statistics for the constant-u0 ensemble,Uses only the 45 constant-u0 simulations and r...
1,EKE evolution revision,NH EKE evolution with initial 30–70°N Eady-gro...,Adds experiment legends and initial midlatitud...
2,Current Figure 9,Cyclone interaction from eddy surface pressure,Combines the hemispheric evolution and the b2n...
3,Current Figure 10,1000-hPa wind evolution around eddy-contour co...,Separates the time-series figure and adds pane...
4,Current Figure 11,Persistent-anticyclone evolution diagnosed fro...,"Uses eddy-SLP contours, improved spacing, corr..."
5,Current Figure 12,High-pressure eddy-SLP Hovmöller diagrams,"Uses 2-hPa intervals, a ±2-hPa white center, ±..."
6,Heatmap presentation revision,Eddy-SLP cyclone counts and anticyclone duration,"Applies the final JAMES-style panel labels, re..."
7,SI Figures S9–S10,Initial and growth-phase upper/lower baroclini...,Uses 1000–850 hPa for the lower-layer shading ...
8,SI Figure S11,RWB orientation fractions in the two ensembles,Updates visible terminology to constant u0 and...
9,SI Figures S12–S13,Initial cross sections with corrected colorbars,Repositions colorbars without changing the und...


## Current Figure 6: RWB statistics for the constant-u0 ensemble

Uses only the 45 constant-u0 simulations and removes the in-figure overall title.

- Script: `revision_20260823/artifacts/main_rwb_constant_u0/scripts/replot_figure5_rwb_constant_u0_no_suptitle.py`

![RWB statistics for the constant-u0 ensemble](revision_20260823/artifacts/main_rwb_constant_u0/figures/Figure6_RWB_fraction_constant_u0_no_suptitle.png)

## EKE evolution revision: NH EKE evolution with initial 30–70°N Eady-growth context

Adds experiment legends and initial midlatitude Eady-growth annotations using the NH-consistent diagnostic.

- Script: `revision_20260823/artifacts/main_eke_midlatitude_eady/scripts/plot_figure6_eke_with_midlatitude_eady.py`
- Supporting data: `revision_20260823/artifacts/main_eke_midlatitude_eady/tables/Figure6_initial_midlatitude_Eady_annotations.csv`

![NH EKE evolution with initial 30–70°N Eady-growth context](revision_20260823/artifacts/main_eke_midlatitude_eady/figures/Figure6_EKE_evolution_with_midlatitude_Eady_mean.png)

In [2]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/main_eke_midlatitude_eady/tables/Figure6_initial_midlatitude_Eady_annotations.csv'
pd.read_csv(table_path).head(10)

,case,b,n,s,initial_eady_area_mass_weighted_30_70N_day-1,eady_valid_weight_fraction,eady_domain_latitude_min_deg,eady_domain_latitude_max_deg,eady_pressure_top_hpa,eady_pressure_bottom_hpa
0,BCwave_b1n1s-10,1.0,1,-10,0.278054,1.0,30.5,69.5,10.0,1000.0
1,BCwave_b1n1,1.0,1,0,0.336661,1.0,30.5,69.5,10.0,1000.0
2,BCwave_b1n1s10,1.0,1,10,0.331256,1.0,30.5,69.5,10.0,1000.0
3,BCwave_b1n3s-10,1.0,3,-10,0.200406,1.0,30.5,69.5,10.0,1000.0
4,BCwave_b1n3,1.0,3,0,0.268753,1.0,30.5,69.5,10.0,1000.0
5,BCwave_b1n3s10,1.0,3,10,0.256039,1.0,30.5,69.5,10.0,1000.0
6,BCwave_b1n6s-10,1.0,6,-10,0.158551,1.0,30.5,69.5,10.0,1000.0
7,BCwave_b1n6,1.0,6,0,0.213400,1.0,30.5,69.5,10.0,1000.0
8,BCwave_b1n6s10,1.0,6,10,0.199719,1.0,30.5,69.5,10.0,1000.0
9,BCwave_b15n1s-10,1.5,1,-10,0.345850,1.0,30.5,69.5,10.0,1000.0


## Current Figure 9: Cyclone interaction from eddy surface pressure

Combines the hemispheric evolution and the b2n3s10 local zoom using eddy-SLP contours.

- Script: `revision_20260823/artifacts/main_cyclone_overview/scripts/reorganize_eddy_cyclone_figures.py`

![Cyclone interaction from eddy surface pressure](revision_20260823/artifacts/main_cyclone_overview/figures/Figure9_eddy_cyclone_overview_plus_zoom.png)

## Current Figure 10: 1000-hPa wind evolution around eddy-contour connection

Separates the time-series figure and adds panel-specific legends for s, n, and b.

- Script: `revision_20260823/artifacts/main_cyclone_wind_evolution/scripts/reorganize_eddy_cyclone_figures.py`
- Supporting data: `revision_20260823/artifacts/main_cyclone_wind_evolution/tables/Figure8_eddy_wind_speed_series_standard.csv`

![1000-hPa wind evolution around eddy-contour connection](revision_20260823/artifacts/main_cyclone_wind_evolution/figures/Figure10_eddy_cyclone_wind_evolution_timeseries.png)

In [3]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/main_cyclone_wind_evolution/tables/Figure8_eddy_wind_speed_series_standard.csv'
pd.read_csv(table_path).head(10)

,case,relative_hour,wind_speed_m_s,b,n,s
0,BCwave_b15n1,-27,44.235561,1.5,1,0
1,BCwave_b15n1,-26,45.044438,1.5,1,0
2,BCwave_b15n1,-25,45.070476,1.5,1,0
3,BCwave_b15n1,-24,45.085964,1.5,1,0
4,BCwave_b15n1,-23,45.395618,1.5,1,0
5,BCwave_b15n1,-22,45.688007,1.5,1,0
6,BCwave_b15n1,-21,45.775912,1.5,1,0
7,BCwave_b15n1,-20,45.958159,1.5,1,0
8,BCwave_b15n1,-19,46.121659,1.5,1,0
9,BCwave_b15n1,-18,46.292474,1.5,1,0


## Current Figure 11: Persistent-anticyclone evolution diagnosed from eddy SLP

Uses eddy-SLP contours, improved spacing, corrected sectors, and two high-center labels in the local panels.

- Script: `revision_20260823/artifacts/main_persistent_anticyclone/scripts/plot_figure11_eddy_qc.py`
- Supporting data: `revision_20260823/artifacts/main_persistent_anticyclone/tables/Figure11_eddy_panel_metadata.csv`

![Persistent-anticyclone evolution diagnosed from eddy SLP](revision_20260823/artifacts/main_persistent_anticyclone/figures/Figure11_eddy_persistent_anticyclone.png)

In [4]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/main_persistent_anticyclone/tables/Figure11_eddy_panel_metadata.csv'
pd.read_csv(table_path).head(10)

,panel,day,field,high_center_count,box_bounds
0,a,7,500-hPa omega,1,NaN
1,b,8,500-hPa omega,2,NaN
2,c,9,500-hPa omega,2,NaN
3,d,10,500-hPa omega,2,"100-250E, 35-65N"
4,e,11,500-hPa omega,2,NaN
5,f,12,500-hPa omega,2,"100-250E, 35-65N"
6,g,10,1000-hPa temperature anomaly,2,"100-250E, 35-65N"
7,h,12,1000-hPa temperature anomaly,2,"100-250E, 35-65N"
8,i,14,1000-hPa temperature anomaly,2,"100-250E, 35-65N"


## Current Figure 12: High-pressure eddy-SLP Hovmöller diagrams

Uses 2-hPa intervals, a ±2-hPa white center, ±10-hPa limits, and corrected b-axis labels.

- Script: `revision_20260823/artifacts/main_high_pressure_hovmoller/scripts/replot_figure12_eddy_hovmoller_2hpa_pm10_blabels.py`

![High-pressure eddy-SLP Hovmöller diagrams](revision_20260823/artifacts/main_high_pressure_hovmoller/figures/Figure12_eddy_high_pressure_hovmoller_2hpa_pm10_blabels.png)

## Heatmap presentation revision: Eddy-SLP cyclone counts and anticyclone duration

Applies the final JAMES-style panel labels, removes oversized suptitles, and retains the eddy-field diagnostics.

- Script: `revision_20260823/artifacts/main_heatmaps_james_qc/scripts/replot_heatmaps_james_qc.py`

![Eddy-SLP cyclone counts and anticyclone duration](revision_20260823/artifacts/main_heatmaps_james_qc/figures/Figure6_eddy_slp_coalescence_heatmaps.png)

![Eddy-SLP cyclone counts and anticyclone duration](revision_20260823/artifacts/main_heatmaps_james_qc/figures/Figure13_eddy_anticyclone_overlap_duration.png)

## SI Figures S9–S10: Initial and growth-phase upper/lower baroclinicity profiles

Uses 1000–850 hPa for the lower-layer shading and labels the ensembles constant u0 and constant Umax = 30 m s−1.

- Script: `revision_20260823/artifacts/si_s9_s10_upper_lower/scripts/make_upper_lower_baroclinicity_figures.py`
- Supporting data: `revision_20260823/artifacts/si_s9_s10_upper_lower/tables/initial_vertical_profiles.nc`
- Supporting data: `revision_20260823/artifacts/si_s9_s10_upper_lower/tables/growth_stage_flux_profiles.nc`
- Supporting data: `revision_20260823/artifacts/si_s9_s10_upper_lower/tables/upper_lower_baroclinicity_profile_statistics.csv`

![Initial and growth-phase upper/lower baroclinicity profiles](revision_20260823/artifacts/si_s9_s10_upper_lower/figures/initial_vertical_structure_standard_vs_u30_1000_850.png)

![Initial and growth-phase upper/lower baroclinicity profiles](revision_20260823/artifacts/si_s9_s10_upper_lower/figures/eddy_flux_vertical_structure_standard_vs_u30_1000_850.png)

In [5]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/si_s9_s10_upper_lower/tables/upper_lower_baroclinicity_profile_statistics.csv'
pd.read_csv(table_path).head(10)

,figure,ensemble,b,metric,pressure_hpa,mean,bootstrap_ci_low,bootstrap_ci_high,n_ns_groups,group_mean_peak_abs_pressure_hpa,group_mean_peak_value
0,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,10.0,0.036216,0.033501,0.038932,15,NaN,NaN
1,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,50.0,-0.045168,-0.048556,-0.041782,15,NaN,NaN
2,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,100.0,-0.296261,-0.318470,-0.273999,15,NaN,NaN
3,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,150.0,-0.756635,-0.813509,-0.699458,15,NaN,NaN
4,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,200.0,-1.249329,-1.343454,-1.154338,15,NaN,NaN
5,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,250.0,-1.572376,-1.691026,-1.452344,15,NaN,NaN
6,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,300.0,-1.583437,-1.702969,-1.462444,15,NaN,NaN
7,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,400.0,-0.921853,-0.991289,-0.851848,15,NaN,NaN
8,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,500.0,0.146914,0.135807,0.157978,15,NaN,NaN
9,initial_vertical_structure_standard_vs_u30,standard,1.0,initial_vertical_wind_shear,600.0,1.276655,1.179446,1.374365,15,NaN,NaN


## SI Figure S11: RWB orientation fractions in the two ensembles

Updates visible terminology to constant u0 and constant Umax = 30 m s−1.

- Script: `revision_20260823/artifacts/si_s11_rwb_two_ensembles/scripts/make_original_style_comparison_figures.py`
- Supporting data: `revision_20260823/artifacts/si_s11_rwb_two_ensembles/tables/controlled_rwb_direct_counts_all45.csv`

![RWB orientation fractions in the two ensembles](revision_20260823/artifacts/si_s11_rwb_two_ensembles/figures/Figure5_original_vs_controlled_same_style.png)

In [6]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/si_s11_rwb_two_ensembles/tables/controlled_rwb_direct_counts_all45.csv'
pd.read_csv(table_path).head(10)

,case,b,n,s,cyclonic,anticyclonic,total
0,BCwave_b1_u30_n1_s-10,1.0,1,-10,1290,391,1681
1,BCwave_b1_u30_n1_s-5,1.0,1,-5,1561,690,2251
2,BCwave_b1_u30_n1_s0,1.0,1,0,451,939,1390
3,BCwave_b1_u30_n1_s5,1.0,1,5,244,714,958
4,BCwave_b1_u30_n1_s10,1.0,1,10,111,715,826
5,BCwave_b1_u30_n3_s-10,1.0,3,-10,1591,28,1619
6,BCwave_b1_u30_n3_s-5,1.0,3,-5,1296,194,1490
7,BCwave_b1_u30_n3_s0,1.0,3,0,343,331,674
8,BCwave_b1_u30_n3_s5,1.0,3,5,204,375,579
9,BCwave_b1_u30_n3_s10,1.0,3,10,153,672,825


## SI Figures S12–S13: Initial cross sections with corrected colorbars

Repositions colorbars without changing the underlying initial-state or conversion diagnostics.

- Script: `revision_20260823/artifacts/si_s12_s13_colorbars/scripts/fix_figure_s12_colorbars.py`
- Script: `revision_20260823/artifacts/si_s12_s13_colorbars/scripts/fix_figure_s13_colorbar.py`

![Initial cross sections with corrected colorbars](revision_20260823/artifacts/si_s12_s13_colorbars/figures/Figure_S12_initial_cross_sections_b_variation_colorbar_fixed.png)

![Initial cross sections with corrected colorbars](revision_20260823/artifacts/si_s12_s13_colorbars/figures/Figure_S13_initial_cross_sections_n_s_variations_colorbar_fixed.png)

## SI Figure S14: Constant-u0 and constant-Umax relationship comparison

Provides the two-ensemble predictor/response comparison with consistent terminology.

- Script: `revision_20260823/artifacts/si_s14_two_ensemble_correlations/scripts/build_figure_s14_two_ensembles.py`
- Supporting data: `revision_20260823/artifacts/si_s14_two_ensemble_correlations/tables/Figure_S14_two_ensemble_spearman_long.csv`
- Supporting data: `revision_20260823/artifacts/si_s14_two_ensemble_correlations/tables/Figure_S14_constant_u0_correlation_matrix.csv`
- Supporting data: `revision_20260823/artifacts/si_s14_two_ensemble_correlations/tables/Figure_S14_constant_Umax_30_ms_correlation_matrix.csv`

![Constant-u0 and constant-Umax relationship comparison](revision_20260823/artifacts/si_s14_two_ensemble_correlations/figures/Figure_S14_constant_u0_vs_constant_Umax.png)

In [7]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/si_s14_two_ensemble_correlations/tables/Figure_S14_two_ensemble_spearman_long.csv'
pd.read_csv(table_path).head(10)

,ensemble,predictor,predictor_label,response,response_label,n,spearman_rho,p_value
0,constant_u0,low_level_eady_max_day-1,Low-level Eady growth,peak_eke_m2s-2,Peak 300-hPa EKE,45,0.869433,9.387080e-15
1,constant_u0,low_level_eady_max_day-1,Low-level Eady growth,post_onset_awb_fraction,Post-onset AWB fraction,44,0.746141,6.130231e-09
2,constant_u0,low_level_eady_max_day-1,Low-level Eady growth,eddy_minus10_coalescence_count,−10-hPa eddy-SLP coalescence count,45,0.757842,1.664939e-09
3,constant_u0,low_level_eady_max_day-1,Low-level Eady growth,max_anticyclone_overlap_lifetime_h,Maximum anticyclone- overlap duration,45,0.868092,1.153115e-14
4,constant_u0,u_max_initial_ms,Initial Umax,peak_eke_m2s-2,Peak 300-hPa EKE,45,0.934651,6.313340e-21
5,constant_u0,u_max_initial_ms,Initial Umax,post_onset_awb_fraction,Post-onset AWB fraction,44,0.390550,8.766135e-03
6,constant_u0,u_max_initial_ms,Initial Umax,eddy_minus10_coalescence_count,−10-hPa eddy-SLP coalescence count,45,0.632831,3.096008e-06
7,constant_u0,u_max_initial_ms,Initial Umax,max_anticyclone_overlap_lifetime_h,Maximum anticyclone- overlap duration,45,0.636439,2.612032e-06
8,constant_u0,rossby_penetration_theory_km,Rossby penetration H_R = fL/N,peak_eke_m2s-2,Peak 300-hPa EKE,45,0.486166,7.095081e-04
9,constant_u0,rossby_penetration_theory_km,Rossby penetration H_R = fL/N,post_onset_awb_fraction,Post-onset AWB fraction,44,0.486861,8.042842e-04


## SI Figure S15: Standard-ensemble jet and low-level-baroclinicity adjustment

Combines representative cross sections with independently verified Day-8/Day-15 statistics for all 45 standard cases.

- Script: `revision_20260823/artifacts/si_s15_jet_adjustment/scripts/make_figure_S15_standard_ensemble_update.py`
- Supporting data: `revision_20260823/artifacts/si_s15_jet_adjustment/tables/Figure_S15_plotted_values_all45.csv`

![Standard-ensemble jet and low-level-baroclinicity adjustment](revision_20260823/artifacts/si_s15_jet_adjustment/figures/Figure_S15_standard_ensemble_jet_adjustment.png)

In [8]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/si_s15_jet_adjustment/tables/Figure_S15_plotted_values_all45.csv'
pd.read_csv(table_path).head(10)

,case,b,n,s,initial_time_hour,initial_time_day,initial_u_max_300_ms,initial_jet_latitude_300_degN,initial_max_abs_dtheta850_dy_K_per_1000km,initial_max_abs_dtheta850_dy_latitude_degN,...,day8_delta_max_abs_dtheta850_dy_K_per_1000km,day15_delta_u_max_300_ms,day15_delta_jet_latitude_300_deg,day15_delta_max_abs_dtheta850_dy_K_per_1000km,analysis_max_abs_u_max_300_change_ms,analysis_max_abs_jet_latitude_300_change_deg,fixed_b_n_group,day15_jet_latitude_strict_s_order_retained,fixed_n_s_group,day15_u_max_300_strict_b_order_retained
0,BCwave_b1n1s-10,1.0,1,-10,1.0,0.041667,9.922583,34.5,10.163475,44.5,...,0.008590,0.280697,1.0,0.739289,0.280697,1.0,"b=1, n=1",True,"n=1, s=-10",True
1,BCwave_b1n1s-5,1.0,1,-5,1.0,0.041667,9.923443,39.5,11.455005,48.5,...,0.014835,0.038600,1.0,0.314092,0.108895,1.0,"b=1, n=1",True,"n=1, s=-5",True
2,BCwave_b1n1,1.0,1,0,1.0,0.041667,9.924018,44.5,12.657235,53.5,...,0.038399,0.493165,5.0,0.752216,0.493165,5.0,"b=1, n=1",True,"n=1, s=0",True
3,BCwave_b1n1s5,1.0,1,5,1.0,0.041667,9.925054,49.5,13.718158,54.5,...,-0.098444,1.146592,3.0,-0.823885,1.146592,4.0,"b=1, n=1",True,"n=1, s=5",True
4,BCwave_b1n1s10,1.0,1,10,1.0,0.041667,9.925867,54.5,14.708152,60.5,...,-0.366271,0.961193,4.0,-0.747840,0.961193,5.0,"b=1, n=1",True,"n=1, s=10",True
5,BCwave_b1n3s-10,1.0,3,-10,1.0,0.041667,9.907720,34.5,9.261487,38.5,...,-0.009896,0.123727,0.0,-1.225827,0.123727,0.0,"b=1, n=3",True,"n=3, s=-10",True
6,BCwave_b1n3s-5,1.0,3,-5,1.0,0.041667,9.909569,39.5,10.580532,43.5,...,-0.039009,0.224652,0.0,-1.765556,0.224652,0.0,"b=1, n=3",True,"n=3, s=-5",True
7,BCwave_b1n3,1.0,3,0,1.0,0.041667,9.910440,44.5,11.988714,48.5,...,-0.215292,0.012332,1.0,-2.171177,0.106235,1.0,"b=1, n=3",True,"n=3, s=0",True
8,BCwave_b1n3s5,1.0,3,5,1.0,0.041667,9.912292,49.5,13.254177,52.5,...,-0.464494,-0.118736,3.0,-2.698005,0.228568,3.0,"b=1, n=3",True,"n=3, s=5",True
9,BCwave_b1n3s10,1.0,3,10,1.0,0.041667,9.912517,54.5,14.317131,56.5,...,-0.710746,0.521244,5.0,-3.174637,0.521244,5.0,"b=1, n=3",True,"n=3, s=10",True


## SI Figure S19: Primary/secondary cyclone-interaction atlas

Reformats the 44-case atlas from four to eight rows per page and separates the colorbar from the lowest row.

- Script: `revision_20260823/artifacts/si_s19_primary_secondary_atlas/scripts/build_primary_secondary_atlas_8rows.py`
- Supporting data: `revision_20260823/artifacts/si_s19_primary_secondary_atlas/tables/Figure_A1_primary_secondary_atlas_8rows_crosswalk.csv`

![Primary/secondary cyclone-interaction atlas](revision_20260823/artifacts/si_s19_primary_secondary_atlas/figures/Figure_A1_primary_secondary_8rows_page_01_of_05.png)

![Primary/secondary cyclone-interaction atlas](revision_20260823/artifacts/si_s19_primary_secondary_atlas/figures/Figure_A1_primary_secondary_8rows_page_02_of_05.png)

![Primary/secondary cyclone-interaction atlas](revision_20260823/artifacts/si_s19_primary_secondary_atlas/figures/Figure_A1_primary_secondary_8rows_page_03_of_05.png)

![Primary/secondary cyclone-interaction atlas](revision_20260823/artifacts/si_s19_primary_secondary_atlas/figures/Figure_A1_primary_secondary_8rows_page_04_of_05.png)

![Primary/secondary cyclone-interaction atlas](revision_20260823/artifacts/si_s19_primary_secondary_atlas/figures/Figure_A1_primary_secondary_8rows_page_05_of_05.png)

In [9]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/si_s19_primary_secondary_atlas/tables/Figure_A1_primary_secondary_atlas_8rows_crosswalk.csv'
pd.read_csv(table_path).head(10)

,case,b,n,s,mode,threshold_hpa,time_index,time_hour,target_id,n_source_ids,...,track_target_lat,track_target_lon,selection_rule,available_minus24,available_plus24,available_plus48,atlas_event_id,page,row_on_page,hemisphere
0,BCwave_b1n1s-10,1.0,1,-10,full,990.0,336,337,1.0,2.0,...,47.945884,94.314172,first NH primary-track coalescence; exact 2-to...,True,False,False,1,1,1,NH
1,BCwave_b15n1s-10,1.5,1,-10,full,990.0,316,317,1.0,2.0,...,50.946414,106.797503,first NH primary-track coalescence; exact 2-to...,True,True,False,2,1,2,NH
2,BCwave_b2n1s-10,2.0,1,-10,full,990.0,310,311,1.0,2.0,...,52.907899,116.794383,first NH primary-track coalescence; exact 2-to...,True,True,True,3,1,3,NH
3,BCwave_b1n1s-5,1.0,1,-5,full,990.0,311,312,1.0,2.0,...,53.121982,95.745445,first NH primary-track coalescence; exact 2-to...,True,True,True,4,1,4,NH
4,BCwave_b15n1s-5,1.5,1,-5,full,990.0,284,285,2.0,2.0,...,55.671600,108.266515,first NH primary-track coalescence; exact 2-to...,True,True,True,5,1,5,NH
5,BCwave_b15n3s-5,1.5,3,-5,full,990.0,328,329,2.0,2.0,...,50.716178,112.414006,first NH primary-track coalescence; exact 2-to...,True,True,False,6,1,6,NH
6,BCwave_b15n6s-5,1.5,6,-5,full,990.0,356,357,1.0,2.0,...,49.575580,112.284844,first NH primary-track coalescence; exact 2-to...,True,False,False,7,1,7,NH
7,BCwave_b2n1s-5,2.0,1,-5,full,990.0,273,274,1.0,2.0,...,57.153781,118.378346,first NH primary-track coalescence; exact 2-to...,True,True,True,8,1,8,NH
8,BCwave_b2n3s-5,2.0,3,-5,full,990.0,320,321,1.0,2.0,...,51.823392,119.081308,first NH primary-track coalescence; exact 2-to...,True,True,False,9,2,1,NH
9,BCwave_b2n6s-5,2.0,6,-5,full,990.0,349,350,1.0,2.0,...,50.323801,118.333600,first NH primary-track coalescence; exact 2-to...,True,False,False,10,2,2,NH


## SI Figures S20–S21: Eddy-SLP anticyclone tracking and definition sensitivity

Recomputes objects, tracks, lifetimes, mobility, and sensitivity statistics from eddy surface pressure rather than full SLP.

- Script: `revision_20260823/artifacts/si_s20_s21_eddy_slp/scripts/recompute_figures_s20_s21_eddy_slp.py`
- Supporting data: `revision_20260823/artifacts/si_s20_s21_eddy_slp/tables/S20_case_level_results.csv`
- Supporting data: `revision_20260823/artifacts/si_s20_s21_eddy_slp/tables/S21_grouped_results.csv`
- Supporting data: `revision_20260823/artifacts/si_s20_s21_eddy_slp/tables/S21_definition_audit.csv`
- Supporting data: `revision_20260823/artifacts/si_s20_s21_eddy_slp/tables/S21_baseline_validation.csv`

![Eddy-SLP anticyclone tracking and definition sensitivity](revision_20260823/artifacts/si_s20_s21_eddy_slp/figures/Figure_S20_eddy_slp_lifetime_centroid_speed.png)

![Eddy-SLP anticyclone tracking and definition sensitivity](revision_20260823/artifacts/si_s20_s21_eddy_slp/figures/Figure_S21_eddy_slp_definition_sensitivity.png)

In [10]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/si_s20_s21_eddy_slp/tables/S20_case_level_results.csv'
pd.read_csv(table_path).head(10)

,case,b,n,s,definition_id,method,eddy_threshold_hpa,criterion_value,minimum_area_km2,minimum_duration_h,...,mean_qualifying_object_area_km2,maximum_qualifying_object_area_km2,maximum_peak_eddy_slp_anomaly_hpa,start_hour,end_hour,mean_centroid_path_speed_m_s-1,net_centroid_speed_m_s-1,mean_step_displacement_km,mean_normalized_step,source_file
0,BCwave_b1n1s-10,1.0,1,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,6.043859e+05,1.176950e+06,12.142799,334,360,4.887906,4.109438,17.596460,0.047200,/data/keeling/a/mingfei5/a/data/original/BCwav...
1,BCwave_b1n3s-10,1.0,3,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,NaN,NaN,NaN,0,0,NaN,NaN,NaN,NaN,/data/keeling/a/mingfei5/a/data/original/BCwav...
2,BCwave_b1n6s-10,1.0,6,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,NaN,NaN,NaN,0,0,NaN,NaN,NaN,NaN,/data/keeling/a/mingfei5/a/data/original/BCwav...
3,BCwave_b15n1s-10,1.5,1,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,2.243598e+06,4.636271e+06,17.603288,264,360,6.799192,6.006131,24.477091,0.036398,/data/keeling/a/mingfei5/a/data/original/BCwav...
4,BCwave_b15n3s-10,1.5,3,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,2.357432e+05,3.752464e+05,11.273686,351,360,7.715414,5.819777,27.775490,0.104956,/data/keeling/a/mingfei5/a/data/original/BCwav...
5,BCwave_b15n6s-10,1.5,6,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,NaN,NaN,NaN,0,0,NaN,NaN,NaN,NaN,/data/keeling/a/mingfei5/a/data/original/BCwav...
6,BCwave_b2n1s-10,2.0,1,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,3.155493e+06,7.067518e+06,20.237143,247,344,18.674996,4.918741,67.229986,0.078251,/data/keeling/a/mingfei5/a/data/original/BCwav...
7,BCwave_b2n3s-10,2.0,3,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,8.642624e+05,1.262122e+06,12.093832,302,360,7.095416,5.977289,25.543499,0.056544,/data/keeling/a/mingfei5/a/data/original/BCwav...
8,BCwave_b2n6s-10,2.0,6,-10,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,NaN,NaN,NaN,0,0,NaN,NaN,NaN,NaN,/data/keeling/a/mingfei5/a/data/original/BCwav...
9,BCwave_b1n1s-5,1.0,1,-5,S20_plus10_overlap0.3_identity,independent_permissive_overlap_identity,10,0.3,100000.0,0,...,1.505073e+06,2.901241e+06,16.681572,280,360,5.390571,4.693614,19.406055,0.037720,/data/keeling/a/mingfei5/a/data/original/BCwav...


## Table S3: Eddy-SLP persistence, size, intensity, and centroid mobility

Recomputes the fixed n=1, s=+10° table consistently with the eddy-SLP tracker used for Figures S20–S21.

- Script: `revision_20260823/artifacts/table_s3_eddy_slp/scripts/recompute_table_s3_eddy_slp.py`
- Supporting data: `revision_20260823/artifacts/table_s3_eddy_slp/tables/Table_S3_eddy_slp_recomputed.csv`
- Supporting data: `revision_20260823/artifacts/table_s3_eddy_slp/tables/Table_S3_old_full_field_vs_new_eddy_slp.csv`
- Supporting data: `revision_20260823/artifacts/table_s3_eddy_slp/tables/Table_S3_validation_against_Figure_S20.csv`

In [11]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/table_s3_eddy_slp/tables/Table_S3_eddy_slp_recomputed.csv'
pd.read_csv(table_path).head(10)

,case,b,n,s,display_case,source_file,eddy_slp_definition,eddy_threshold_hpa,minimum_area_km2,identity_overlap_threshold,...,mean_area_1e6_km2,maximum_area_km2,peak_eddy_slp_anomaly_hpa,path_speed_m_s-1,net_speed_m_s-1,maximum_absolute_zonal_mean_residual_hpa,number_continuous_72h_windows,minimum_72h_net_speed_m_s-1,median_72h_net_speed_m_s-1,maximum_72h_net_speed_m_s-1
0,BCwave_b1n1s10,1.0,1,10,b1n1s10,/data/keeling/a/mingfei5/a/data/original/BCwav...,PRESsfc minus instantaneous zonal mean at each...,10.0,100000.0,0.3,...,2.193247,4.427112e+06,23.594321,7.412690,2.573614,1.030761e-13,71,0.412451,3.374568,4.644217
1,BCwave_b15n1s10,1.5,1,10,b1.5n1s10,/data/keeling/a/mingfei5/a/data/original/BCwav...,PRESsfc minus instantaneous zonal mean at each...,10.0,100000.0,0.3,...,5.524811,1.396696e+07,28.409881,19.733658,4.124131,1.568373e-13,99,2.197490,5.211814,10.626183
2,BCwave_b2n1s10,2.0,1,10,b2n1s10,/data/keeling/a/mingfei5/a/data/original/BCwav...,PRESsfc minus instantaneous zonal mean at each...,10.0,100000.0,0.3,...,7.322543,2.089106e+07,69.323566,17.833589,2.009665,2.093859e-13,115,5.569305,10.200701,14.905919


## EP-flux supplementary diagnostic: EP-flux evolution during the baroclinic-wave life cycle

Uses pressure-coordinate QG EP flux, corrected log-pressure presentation, less-dense lower-level vectors, and common scales.

- Script: `revision_20260823/artifacts/ep_flux_evolution/scripts/compute_r1_6_ep_flux_evolution.py`

![EP-flux evolution during the baroclinic-wave life cycle](revision_20260823/artifacts/ep_flux_evolution/figures/r1_6_ep_flux_evolution.png)

## Initial-PV supplementary diagnostic: Initial PV and dynamical-tropopause structure across b, n, and s

Expands the original three-panel latitude-shift comparison to a controlled 3×3 b/n/s comparison with shared levels and labeled theta contours.

- Script: `revision_20260823/artifacts/initial_pv_bns/scripts/compute_r1_11_initial_pv_cross_sections_bns.py`

![Initial PV and dynamical-tropopause structure across b, n, and s](revision_20260823/artifacts/initial_pv_bns/figures/r1_11_initial_pv_cross_sections_bns.png)

## Section 3.2 consolidated diagnostic: Vertical profiles, Eady growth, NH EKE, and mature-stage eddy fluxes

Consolidates four two-column comparisons with consistent panel numbering and constant-u0/constant-Umax terminology.

- Script: `revision_20260823/artifacts/section32_combined/scripts/make_section32_figures.py`
- Supporting data: `revision_20260823/artifacts/section32_combined/tables/initial_profiles_at_jet_core.csv`
- Supporting data: `revision_20260823/artifacts/section32_combined/tables/eke_nh_area_mass_weighted_timeseries_all90.csv`
- Supporting data: `revision_20260823/artifacts/section32_combined/tables/eddy_flux_profiles_50_80pct_peak_eke_all90.csv`

![Vertical profiles, Eady growth, NH EKE, and mature-stage eddy fluxes](revision_20260823/artifacts/section32_combined/figures/section32_four_part_combined_8panels.png)

In [12]:
table_path = ROOT / 'publication_notebooks' / 'revision_20260823/artifacts/section32_combined/tables/initial_profiles_at_jet_core.csv'
pd.read_csv(table_path).head(10)

,ensemble,case,b,n,s,pressure_hpa,initial_zonal_wind_at_jet_core_m_s-1,initial_eady_growth_rate_day-1,initial_jet_core_latitude_deg
0,standard,BCwave_b1n1s-10,1.0,1,-10,10.0,0.015421,0.004778,34.5
1,standard,BCwave_b1n1s-10,1.0,1,-10,50.0,0.027734,0.005361,34.5
2,standard,BCwave_b1n1s-10,1.0,1,-10,100.0,0.405482,0.041259,34.5
3,standard,BCwave_b1n1s-10,1.0,1,-10,150.0,1.810515,0.137303,34.5
4,standard,BCwave_b1n1s-10,1.0,1,-10,200.0,4.226558,0.250454,34.5
5,standard,BCwave_b1n1s-10,1.0,1,-10,250.0,7.119331,0.323045,34.5
6,standard,BCwave_b1n1s-10,1.0,1,-10,300.0,9.922583,0.333342,34.5
7,standard,BCwave_b1n1s-10,1.0,1,-10,400.0,13.908433,0.201641,34.5
8,standard,BCwave_b1n1s-10,1.0,1,-10,500.0,15.054868,0.032719,34.5
9,standard,BCwave_b1n1s-10,1.0,1,-10,600.0,13.803334,0.281744,34.5


## Reproduction and provenance

The exact scripts and compact supporting tables are stored under `publication_notebooks/revision_20260823/artifacts/`. Large raw SHiELD NetCDF files are intentionally not duplicated in the Git repository. Scripts retain the original Keeling data dependencies documented in the project README and individual analysis reports. `SHA256SUMS` records every file included in this revision bundle.